src/app/fdctl/config/default.toml

```diff
@@ -1,585 +1,663 @@
+# will be permissioned as read/write for this user, irrespective of
+# which user is performing the configuration.
+#
+# Firedancer requires nothing from this user, and it should be as
+# minimally permissioned as is possible.  It is suggested to run
+# Firedancer as a separate user from other proecssses on the machine so
+# that they cannot attempt to send signals, ptrace, or otherwise
+# interfere with the process.  You should under no circumstances use a
+# superuser or privileged user here, and Firedancer will not allow you
+# to use the root user.
+#
+# When running in a development environment (with `fddev`), Firedancer
+# will automatically determine a user to run as if none is provided.  By
+# default, it will use the terminal user which is starting the program.
+# If running as sudo, the terminal user is not root but the user which
+# invoked sudo.
 user = ""
 
 # Absolute directory path to place scratch files used during setup and
-# operation. Information about the running process is also placed here in a
-# `config.cfg` file so monitoring and debugging tools can find it and connect
-# automatically.
-#
-# In future, Firedancer will also page the Solana accounts database to disk in
-# this directory but this is not currently needed, as the Solana Labs validator
-# still manages the accounts database.
+# operation.  The ledger and accounts databases will also be placed in
+# here by default, although that is overridable by other options.
 #
-# Two substitutions will be performed on this string. If "{user}" is present it
-# will be replaced with the user running Firedancer, as above, and "{name}" will
-# be replaced with the name of the Firedancer instance.
+# Two substitutions will be performed on this string.  If "{user}" is
+# present it will be replaced with the user running Firedancer, as
+# above, and "{name}" will be replaced with the name of the Firedancer
+# instance.
 scratch_directory = "/home/{user}/.firedancer/{name}"
 
 # Port range used for various incoming network listeners, in the form
-# `<MIN_PORT>-<MAX_PORT>` inclusive. The range used includes min, but not max
-# [min, max). Ports are used for receiving transactions and votes from clients
-# and other validators. This option is passed to the Solana Labs client with the
-# `--dynamic-port-range` argument.
+# `<MIN_PORT>-<MAX_PORT>` inclusive.  The range used includes min, but
+# not max [min, max).  Ports are used for receiving transactions and
+# votes from clients and other validators.
+#
+# For Firedancer, ports are assigned statically in later parts of this
+# configuration file, and this option is passed to the Solana Labs
+# client with the `--dynamic-port-range` argument.  Solana Labs will use
+# this to determine port locations for services not yet rewritten as
+# part of Firedancer, including gossip and RPC.  This port range should
+# NOT overlap with any of the static ports used by Firedancer below.
 dynamic_port_range = "8000-10000"
```

Comment:
We should make the defaults not overlap. We're using 9001, 9007 for quic. Maybe make this 8500-8600 or something? I don't think it needs to be a huge range.